content/en/docs/tasks/administer-cluster/dns-debugging-resolution.md

```diff
@@ -267,6 +267,10 @@ some of those settings will be lost. As a partial workaround, the node can run
 `dnsmasq` which will provide more `nameserver` entries, but not more `search`
 entries. You can also use kubelet's `--resolv-conf` flag.
 
+{{< note >}}
+6 DNS `search` records limit has been removed in glibc package versions at or exceeding glibc-2.17-222, refer to [this](https://access.redhat.com/solutions/58028) for the details.
+{{< /note >}}
+
```

Comment:
@Colstuwjx I recommend rewording [lines 261-268](https://github.com/kubernetes/website/pull/18913/files#diff-108b8d55763a7f2d881b894d15e12901R261-R268); you might be able to avoid having a note.

You can describe the situation with glibc as historical, now that it's fixed upstream.

Reply:
@sftim  I revised the sentence, thanks for your suggestion!